fsd_utils/mapping/application/application_utils.py

```diff
@@ -40,12 +41,24 @@ def format_answer(answer):
         if "null" in answer:
             return re.sub(r"\s*null\s*,?", "", answer)
 
+        if isinstance(answer, str):
+            if answer.startswith("http") or answer.startswith("https"):
+                return answer
+            elif "-" in answer:
```

Comment:
Do we always want to replace all hyphens with spaces? The free text ones could have a valid `-` character. Wouldn't we only remove the hyphens for like bullet/radio button answers where the 'answer' is actually a hard coded backend value?

Reply:
So maybe do this conditionally based on field type?

Reply:
I thought of it too but the radio button doesn't seem to have a specific field type so it was kind of tricky. I will defo look into it or should we leave the hyphens as it is for the checkbox, radio fields, etc. It does handle the HTTP response